packages/compass-e2e-tests/tests/database-collections-tab.test.ts

```diff
@@ -179,7 +179,9 @@ describe('Database collections tab', function () {
     const collectionCard = await browser.$(selector);
     await collectionCard.waitForDisplayed();
 
-    // TODO: how do we make sure this is really a collection with a custom collation?
+    await collectionCard
```

Comment:
As a driveby I just made sure the collation collection has the collation badge as a cheap way to try and ensure it really did what we asked for.